routes/api.php

```diff
@@ -18,9 +19,19 @@
 
 Route::post('login', [AuthController::class, 'login']);
 Route::post('register', [AuthController::class, 'register']);
+
 Route::resources([
     'venues' => VenueController::class,
+    'venues.show' => VenueController::class,
     'attributes' => AttributeController::class
 ]);
+
+
+Route::group(['prefix' => 'user', 'middleware' => 'auth:sanctum'], function () {
+    Route::get("", [UserController::class, 'index']);
+    Route::get("/{id}", [UserController::class, 'show']);
+});
+
+Route::post('/venues/rate_venue', [VenueController::class, 'rate_venue']);
```

Comment:
could this be in the user group? user/venues/{id}/rate.

I think it should be authenticated at least.